tests/src/test/scala/com/lightbend/paradox/markdown/SnipDirectiveSpec.scala

```diff
@@ -93,6 +93,19 @@ class SnipDirectiveSpec extends MarkdownBaseSpec {
       |</pre>""")
   }
 
+  it should "not truncate snippets" in {
+    markdown("""@@snip[example.scala](tests/src/test/scala/com/lightbend/paradox/markdown/example.scala) { #multi-indented-example }""") shouldEqual html("""
+      |<pre class="prettyprint">
+      |<code class="language-scala">
+      |object AnotherIndentedExample {
+      |def rendered(): Unit = {
```

Comment:
@ennru Do you mean this, that the inner indentation is lost here?

Reply:
Yeah, I think best would be to base indentation off the indentation of the following non-empty line or something like that rather? But I don't think it is possible to get a solution that works for all use-cases without having some extra flags for deciding if to keep indent level or drop (the imports + main method contents vs "normal" code

Reply:
Not sure I want to jump into trying to fix that here though, I just want to make sure no actual characters except whitespace is truncated for now.

Reply:
Yes, just a note on what this gives. Not cutting text is the best solution for now. @2m and I discussed some interplay of code formatters and this snippet cutting, but we didn't have this case.